src/pybind/mgr/dashboard/services/iscsi_cli.py

```diff
@@ -6,10 +6,13 @@
 
 from mgr_module import CLIReadCommand, CLIWriteCommand
 
-from .iscsi_client import IscsiClient
-from .iscsi_config import IscsiGatewaysConfig, IscsiGatewayAlreadyExists, InvalidServiceUrl, \
-    ManagedByOrchestratorException, IscsiGatewayDoesNotExist
 from ..rest_client import RequestException
+from .iscsi_client import IscsiClient  # pylint: disable=import-error
+from .iscsi_config import InvalidServiceUrl  # pylint: disable=import-error
+from .iscsi_config import IscsiGatewayAlreadyExists  # pylint: disable=import-error
+from .iscsi_config import IscsiGatewayDoesNotExist  # pylint: disable=import-error
+from .iscsi_config import IscsiGatewaysConfig  # pylint: disable=import-error
+from .iscsi_config import ManagedByOrchestratorException  # pylint: disable=import-error
```

Comment:
Need to workaround a bug in one of the tools. Use a separate line for every import because autopep8 or isort creates a malformed line when it optimizes the code.
Before:
```
from .iscsi_client import IscsiClient, InvalidServiceUrl, IscsiGatewayAlreadyExists, IscsiGatewayDoesNotExist, IscsiGatewaysConfig, ManagedByOrchestratorException  # pylint: disable=import-error
``` 
or
```
from .iscsi_client import IscsiClient, InvalidServiceUrl, IscsiGatewayAlreadyExists, \
    IscsiGatewayDoesNotExist, IscsiGatewaysConfig, \
    ManagedByOrchestratorException  # pylint: disable=import-error
```
After:
```
from .iscsi_client import InvalidServiceUrl, IscsiClient          # pylint: disable=import-error
```